packages/repo-doctor/src/reporters/mattermost.js

```diff
@@ -0,0 +1,54 @@
+const fetch = require('node-fetch')
+
+const symbolBySeverity = {
+  warn: '⚠️',
+  success: '✅',
+  error: '🚩',
+  info: 'ℹ️'
+}
+
+class MattermostReporter {
+  constructor() {
+    this.checkEnv()
+    this.messages = []
+  }
+
+  checkEnv() {
+    if (!process.env.MATTERMOST_HOOK) {
+      throw new Error(
+        'To be instantiated, mattermost reporter needs process.env.MATTERMOST_HOOK'
+      )
+    }
+  }
+
+  write(msg) {
+    this.messages.push(msg)
+  }
+
+  async sendAllMessages() {
+    const payload = {
+      channel: '@patrick',
```

Comment:
I guess ideally this would be configurable through a command line arg?

Reply:
In the config I think it would be better : { reporters: { mattermost: { channel: '' }}

Reply:
I added this ability in https://github.com/cozy/cozy-libs/pull/1035/commits/5d9f32a5285534623326029c878f813fd12a9524